### PR TITLE
docs(building-windows): add info about winget

### PR DESCRIPTION
**Pull request by SunsetTechuila**
### What does this PR do?

- [x] Documentation or TypeScript types
- [ ] Code changes

### CODE REVIEW

File: docs/project/building-windows.md [hunk 1]
```diff
@@ -51,7 +51,12 @@ By default, running unverified scripts are blocked.
 ```
 
 - [Visual Studio](https://visualstudio.microsoft.com) with the "Desktop Development with C++" workload.
-  - Install Git and CMake from this installer, if not already installed.
+  - Install Git from this installer, if not already installed.
```
Comment (SunsetTechuila) [inline docs/project/building-windows.md]: cmake is selected for the installation by default for the c++ workload

**Comment by paperdave** [comment]
I'm a bit skeptical to add instructions for winget. I've had situations in the past where `winget` isn't even provided on windows, and sometimes using it on a fresh install would just not work at all. That was a long time ago, and I think winget is in a better place now.

Ive had issues with strawberry perl in the past. Were you able to build WebKit with it?

**Comment by SunsetTechuila** [comment]
> I've had situations in the past where winget isn't even provided on windows

True only for Windows Server, LTSC and LTSB. Scoop is not included in any Windows edition tho

> Were you able to build WebKit with it?

No, but if you tell me how, I will try to

**Comment by paperdave** [comment]
i made some small edits, rolling with the winget instructions as-is

thank you for the pr.

**Pull request merged**